pkgs/applications/audio/zynaddsubfx/mruby-zest/default.nix

```diff
@@ -55,22 +56,21 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "mruby-zest";
-  version = "3.0.5";
+  version = "3.0.6";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = "${pname}-build";
     rev = version;
-    sha256 = "0fxljrgamgz2rm85mclixs00b0f2yf109jc369039n1vf0l5m57d";
+    hash = "sha256-rIb6tQimwrUj+623IU5zDyKNWsNYYBElLQClOsP+5Dc=";
     fetchSubmodules = true;
```

Comment:
Do we replace all submodules? If so we should drop this.

Reply:
As per the comment above `postPatch`, I think we can keep it and simply remove the submodule overrides that are being symlinked in. Builds/runs fine doing so.